api-catalog-ui/frontend/src/components/DetailPage/DetailPage.jsx

```diff
@@ -132,12 +132,7 @@ export default class DetailPage extends Component {
         } = countAdditionalContents(selectedService);
         const onlySwaggerPresent = tutorialsCounter === 0 && videosCounter === 0 && useCasesCounter === 0;
         const showSideBar = false;
-        if (
-            hasTiles &&
-            'customStyleConfig' in tiles[0] &&
-            tiles[0].customStyleConfig &&
-            Object.keys(tiles[0].customStyleConfig).length > 0
-        ) {
+        if (hasTiles && tiles[0]?.customStyleConfig) {
```

Comment:
I think this condition is not equivalent, before it was also verifying if there were keys in `customStyleConfig`

Reply:
you're right, thanks, I will add it back